src/spec/billing_spec.js

```diff
@@ -460,4 +460,29 @@ describe('Billing Page', () => {
     expect(billingPage.getPlanPrice()).not.toBe(planDefaultPrice);
   });
 
+  it('should show correct plan status values', () => {
+
+    // Arrange
+    beginAuthenticatedSession();
+    browser.get('/#/settings/my-plan?plan=PLAN-60K');
+    setupSamplePlanInfoResponse();
+    setupSamplePlansResponse();
+    var billingPage = new BillingPage();
+    var dateConcat = '';
+    var date = new Date();
+    var month = ('0' + (date.getMonth() + 1 + 1)).slice(-2);
+    var year = date.getFullYear();
+    if (month > 12) {
+      month = '01';
+      year = year + 1;
+    }
+    dateConcat = dateConcat.concat(year,'-', month, '-', '01');
```

Comment:
This test, will test a different value each day, I understand that it is difficult to mock the time generator right now. So, it is fine for the moment. But if someone can think in an easy way to hard code the date, I think that it will be nicer an safer.